.github/workflows/python.yml

```diff
@@ -89,6 +89,9 @@ jobs:
         workspaces: python
     - uses: ./.github/workflows/build_linux_wheel
     - uses: ./.github/workflows/run_tests
+    # Make sure wheels are not included in the Rust cache
+    - name: Delete wheels
+      run: rm -rf target/wheels/pylance-*.whl
```

Comment:
is there a reason that not doing `rm -rf target/wheels`?

Reply:
Yeah we could just do that.